From 70907ff44dc835283028533267055ee5b95720b1 Mon Sep 17 00:00:00 2001
From: stephen hemminger <stephen@networkplumber.org>
Date: Mon, 24 Jul 2017 10:57:29 -0700
Subject: [PATCH 43/98] netvsc: include rtnetlink.h

Since these files use rtnl_derefernce make sure and include rtnetlink.h

Signed-off-by: Stephen Hemminger <sthemmin@microsoft.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/hyperv/netvsc_drv.c   |    2 ++
 drivers/net/hyperv/rndis_filter.c |    1 +
 2 files changed, 3 insertions(+)

--- a/drivers/net/hyperv/netvsc_drv.c	2017-10-05 14:20:51.299521898 -0700
+++ b/drivers/net/hyperv/netvsc_drv.c	2017-10-05 14:20:51.295521849 -0700
@@ -33,6 +33,8 @@
 #include <linux/if_vlan.h>
 #include <linux/in.h>
 #include <linux/slab.h>
+#include <linux/rtnetlink.h>
+
 #include <net/arp.h>
 #include <net/route.h>
 #include <net/sock.h>
--- a/drivers/net/hyperv/rndis_filter.c	2017-10-05 14:20:51.299521898 -0700
+++ b/drivers/net/hyperv/rndis_filter.c	2017-10-05 14:20:51.295521849 -0700
@@ -28,6 +28,7 @@
 #include <linux/if_vlan.h>
 #include <linux/nls.h>
 #include <linux/vmalloc.h>
+#include <linux/rtnetlink.h>
 
 #include "hyperv_net.h"
 
